src/components/SidebarButton.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+
+class SidebarButton extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isSidebarOpen: false,
+    };
+  }
+  render() {
+    document.body.className = `${this.state.isSidebarOpen ? "msb-x" : ""}`;
```

Comment:
this is a big NO :)
1. you don't want some sneaky SidebarButton to directly alter other elements outside of it
2. last time, I told you not to use `document` reference 🙃